group-coordinator/src/main/resources/common/message/ConsumerGroupMemberMetadataValue.json

```diff
@@ -35,6 +35,20 @@
     { "name": "RebalanceTimeoutMs", "type": "int32", "versions": "0+", "default": -1,
       "about": "The rebalance timeout" },
     { "name": "ServerAssignor", "versions": "0+", "nullableVersions": "0+", "type": "string",
-      "about": "The server assignor to use; or null if not used." }
+      "about": "The server assignor to use; or null if not used." },
+    { "name": "ClassicMemberMetadata", "versions": "0+", "nullableVersions": "0+", "type": "ClassicMemberMetadata",
+      "about": "The classic member metadata which includes the supported protocols of the consumer if it uses the classic protocol. The metadata is null if the consumer uses the consumer protocol.",
+      "fields": [
+        { "name": "SupportedProtocols", "type": "[]ClassicProtocol", "versions": "0+", "taggedVersions": "0+", "tag": 0,
```

Comment:
I think that we should rather put the tag on the `ClassicMemberMetadata` structure. Is there a reason why you did not put it there?

Reply:
no I just put it here by mistake. Thanks for the catch!